src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -408,6 +396,9 @@ private ModAchievements()
      */
     public static void init()
     {
-        AchievementPage.registerAchievementPage(achievementPageMineColonies);
+        //todo add to an advancement page
+        //(achievementPageMineColonies);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)